stockfish/models.py

```diff
@@ -514,18 +514,28 @@ def get_evaluation(self) -> dict:
             elif splitted_text[0] == "bestmove":
                 return evaluation
 
-    def get_top_moves(self, num_top_moves: int = 5) -> List[dict]:
+    def get_top_moves(
+        self, num_top_moves: int = 5, include_info: bool = False
+    ) -> List[dict]:
         """Returns info on the top moves in the position.
 
         Args:
             num_top_moves:
                 The number of moves to return info on, assuming there are at least
                 those many legal moves.
 
+            include_info:
```

Comment:
Will need to wait for https://github.com/py-stockfish/stockfish/pull/14 to add this to README.